source/options.tsx

```diff
@@ -165,8 +167,8 @@ function summaryHandler(event: delegate.Event<MouseEvent>): void {
 	const toggle = feature.querySelector('input.screenshot-toggle')!;
 	toggle.checked = !toggle.checked;
 
-	// Lazy-load image
-	const screenshot = feature.querySelector('img.screenshot')!;
+	// Lazy-load image/video
+	const screenshot = feature.querySelector<HTMLImageElement | HTMLVideoElement>('.screenshot')!;
 	screenshot.src = screenshot.dataset.src!;
```

Comment:
Do videos work in image tags in every browser?

Reply:
Maybe you forgot to read the changes above...

Reply:
Ooh, iyaa